Nodejs/Product/Nodejs/Repl/NpmReplCommand.cs

```diff
@@ -118,7 +108,7 @@ class NpmReplCommand : IReplCommand {
             // In case someone copies filename
             string projectDirectoryPath = File.Exists(projectPath) ? Path.GetDirectoryName(projectPath) : projectPath;
             
-            if (!isGlobalCommand && !(Directory.Exists(projectDirectoryPath) && File.Exists(Path.Combine(projectDirectoryPath, "package.json")))) {
+            if (!isGlobalCommand && !Directory.Exists(projectDirectoryPath)) {
```

Comment:
in the event of `.npm init`, we should warn people and ask them to run `.npm init -y` instead which will accept all the defaults.